extensions.json

```
@@ -125,6 +125,11 @@
       "repository": "https://github.com/CoenraadS/Bracket-Pair-Colorizer-2",
       "version": "0.1.4"
     },
+    {
+      "id": "csstools.postcss",
+      "repository": "https://github.com/csstools/postcss-language",
+      "version": "0.27.0"
```

Since are no dedicated release tags or branch, I think we can publish straight from the default branch without pinning a specific version.

```suggestion
      "repository": "https://github.com/csstools/postcss-language"
```